back/app/services/input_status_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class InputStatusService
+  attr_reader :input_status
+
+  def initialize(input_status)
+    @input_status = input_status
+  end
+
+  def can_transition_manually?
+    case input_status.participation_method
+    when 'ideation'
+      true
+    when 'proposals'
+      !input_status.automatic?
+    else
+      false
+    end
+  end
+
```

Comment:
Should we delegate this to the participation method?

Reply:
@kogre Ideally yes, but I assumed that this would be temporary code. The idea statuses feel less related to the participation context (and more to the consultation level), and when we would have a unified input status system, we will no longer be able to support this requirement.

However, we currently have this requirement that the proposed status can be assigned for ideation but not for proposals. Perhaps we could change the requirement instead so that the logic becomes more unified?